src/helpers/parseIpfs.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios'
+import Web3Utils from 'web3-utils'
+
+// default endpoints to be queried
+const ipfsEndpoints = ['https://ipfs.autark.xyz:5001/api/v0']
```

Comment:
I like this feature, but I don't think it is a good practice to hardcode the autark ipfs node here.

Reply:
I was using that for my own tests, but Aragon One is welcome to add the api endpoints they maintain. I just left it in because @sohkai  usually puts together suggested changes for stuff like this, and this way it passes tests upon submission